Akmal

khikmatullaev opened the pull request:
Pull Request for Issue # .

### Summary of Changes



### Testing Instructions



### Expected result



### Actual result



### Documentation Changes Required

khikmatullaev committed:
Adding Trailing slash, making lower case, removing double slash

Adding Trailing slash
making lower case
removing double slash

khikmatullaev committed:
NONSEF URL to SEF URL

NonSEF url redirection
Case /index.php/ - check if ideas could cause problems there
Case SWITCH NONSEF to SEF
Redirect 301 SEF-URL (/index.php?com_content?view=)
/menualias/artile[/|.html]

dgrammatiko commented:
@khikmatullaev please use strict comparators (e.g. === instead of ==)

zero-24 commented:
@khikmatullaev please also check the issues that drone is mention http://213.160.72.75/joomla-projects/gsoc17_improved_routing/38 :)

khikmatullaev committed:
The coding style changed

I have changed == to === and added spaced after the type in descriptions
of functions

khikmatullaev commented:
I have changed all the coding styles

khikmatullaev commented:
What should I change in files?

khikmatullaev committed:
Merge branch 'master' into Akmal
